src/TitleDetails/TitleDetailsContainer.js

```diff
@@ -70,6 +70,34 @@ const TitleDetailsContainer = ({ location, history, mutator, match }) => {
     [titleId],
   );
 
+  const getHoldingsItemsAndPieces = useCallback((holdingId, params = {}) => {
+    const holdingsPieces = mutator.pieces.GET({
+      params: {
+        limit: `${LIMIT_MAX}`,
+        query: `holdingId==${holdingId}`,
+        ...params,
+      },
+      records: undefined,
```

Comment:
is it required?

Reply:
If I don't specify the `records` field as `undefined`, then I get only `items` record, which specified in manifest.
In the case with confirm modal I don't need all items records, only their `totalRecords`.

Reply:
So, I can set `limit` to `1` and get `items` and `totalRecords` data from the request